### DISCUSSION TITLE
Web Vault login page keep loading on iPadOS ( vw version 1.31.0 and above)

### DISCUSSION THREAD

**Discussion by hjh142857** [dani-garcia/vaultwarden#4912] [Q&A] [answered]
After update the version from 1.25.0 to the latest 1.32.0 , when I open the web vault on ipad , it keep loading and no text box to filling in the email address. 
I tested all versions between 1.25.0 and 1.32.0, also with testing version, no proxy and direct to rocket, with safari and chrome on ipad. 
This problem only occurs on 1.31.0 and above. When I visit the web vault on ipad, no request log output in docker logs.  But tested on another iphone with ios17 works well.
Changing the browser seems no effect, due to browsers uses the same kernel on iOS.
About the ipad device, Safari 15.6.4,  WebKit 605.1.15, iPadOS 15.7.2
Seems like caused by this issue #4024 . I don't know if it's possible to make it compatible with older devices.
![Vaultwarden Web](https://github.com/user-attachments/assets/8f487576-2330-45ba-9f75-22d7f7a55167)

**Comment by BlackDex** [comment]
Clear all the cache of the browser and reverse proxy and see what happens. 

If it works on one device it should on others. It might be DNS on the device it self.

It doesn't have anything to do with the issue you linked to. That is about the Websocket connection and can't cause these issues.

**Reply by hjh142857** [reply]
The browser cache has been all cleared before testing. No reverse proxy used, directly connect to docker by ip:port

**Reply by BlackDex** [reply]
Then check what the browsers developer console reports.

**Reply by hjh142857** [reply]
Thanks for your reply. I'm trying to get a mac first to check console output.
 iphone ipad but windows😭

**Reply by BlackDex** [reply]
Also check the logs of Vaultwarden during the moment you try. If there are no logs, then something in between is blocking the connection.

**Reply by hjh142857** [reply]
It's here. vendor.****.js/bitwarden/src/timestamp.ts:95
When visiting, vaultwarden docker log no new output
The following two images show the output displayed when I first open the webpage and then open the developer tools on a Mac. Strangely, if I open the developer tools first and then refresh the webpage, this error does not occur.
![截屏2024-09-02 上午11 51 38](https://github.com/user-attachments/assets/5483405d-44bd-4e7d-a669-db8dfb16ad46)
![截屏2024-09-02 上午11 52 11](https://github.com/user-attachments/assets/cb47a207-6a8a-4daa-9334-3ebde8c78681)

**Reply by BlackDex** [reply]
Sounds like cache then on since way.
Or maybe something in between is modifying the files.

Try to clear the browser cache for that specific site. So, all cookies, local storage etc...

**Reply by hjh142857** [reply]
I'm sure cleared all the browser data. And same situation in new installed Brave、Firefox on ipad.
I think the source of this error is bitwarden's web vault, I just tried vault.bitwarden.com, keep loading in circle too!!

**Reply by hjh142857** [reply, marked as answer]
The issue has been identified; it's a problem with the upstream Bitwarden. Regardless, thank you for your help and effort.
https://github.com/bitwarden/clients/issues/8348
